client/src/Movies/MovieList.js

```diff
@@ -35,7 +36,7 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
-      <h2>{title}</h2>
+      <Link to={`/movies/${movie.id}`}>{title}</Link>
```

Comment:
The Link should encapsulate the entire movie-card. So I would start the link before the <div className="movie-card"> and I would close the link after the closing div tag